src/test/scala/Test.scala

```diff
@@ -24,12 +31,15 @@ s"""|============================
      }
 
      val pdf = {
-       val renderer = new PdfRenderer(classLoader)
+       val renderer = new PdfRenderer(classLoader, rendererWithFonts)
        renderer.toBytes(body)
      }
 
      Files write (`test.pdf`, pdf)
 
      todo
+
+    // Todo Write proper tests please
```

Comment:
Although I admire your intent. Writing tests for something that wraps another library which results in a side effect (pdf) is probably more trouble than it's worth.

No matter what happens, a human needs to check the resulting PDF on every code change or dependency update.

I am however interested in your thoughts on 'proper tests' here.